api.planx.uk/send/uniform.js

```diff
@@ -203,6 +210,11 @@ async function createZip(stringXml, csv, files, sessionId) {
     zip.addLocalFile(csvPath);
     deleteFile(csvPath);
 
+  // build an optional GeoJSON file for validators
+  if (geojson) {
+    const geoBuff = Buffer.from(JSON.stringify(geojson, null, 2));
+    zip.addFile("boundaryGeoJSON.json", geoBuff);
```

Comment:
we'll need to make sure this hardcoded file name is also present in the XML ([see here](https://github.com/theopensystemslab/planx-new/blob/main/editor.planx.uk/src/%40planx/components/Send/uniform/xml.ts#L34-L45)) - I think we forgot to mention that these XML entries are what tell the Uniform DMS to find/download each file from our zip ! without an entry, it's likely we'd successfully send the file, but it wouldn't actually be pulled down alongside other application files. reference "other" should be fine for this first go

Reply:
one more thought that might be interesting to keep thinking about during UAT: I wonder if .geojson file extension versus .json would be more likely to "automatically open" with whatever spatial software councils likely have installed on their computers?